### PR TITLE
not to use absolute url of product

### PR DESCRIPTION
**Pull request by odk211**
I found only link to product is absolute at top page.
I fix it, and it works.
Check please.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://odk211-master.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://odk211-master.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: frontend/app/views/spree/shared/_products.html.erb [hunk 1]
```diff
@@ -19,7 +19,7 @@
 <% if products.any? %>
   <div id="products" class="row" data-hook>
     <% products.each do |product| %>
-      <% url = spree.product_url(product, taxon_id: @taxon.try(:id)) %>
+      <% url = spree.product_path(product, taxon_id: @taxon.try(:id)) %>
```
Comment (BenMorganIO) [inline frontend/app/views/spree/shared/_products.html.erb]: You should probably update the local variable as well. It still reads `url`. Maybe `path` would be better.

**Comment by odk211** [comment]
@BenMorganIO Thanks, I updated the variable name to path.

**Comment by BenMorganIO** [comment]
:+1:

**Comment by odk211** [comment]
Could you merge it ?

**Comment by BenMorganIO** [comment]
@odk211 @JDutil has the power.

**Comment by odk211** [comment]
@BenMorganIO Thanks!
@JDutil I rebase it, Could you check it? when you have time.

**Comment by JDutil** [comment]
The full path is being used for the http://schema.org/Product seo markup.

**Comment by odk211** [comment]
@JDutil Thanks, I got it.

**Pull request closed without merging**